README.md

```diff
@@ -50,6 +50,37 @@ $ docker-compose -f docker/docker-compose.yml up --build
 Then you'll be able to access the UI from http://localhost:13000 and
 the API from the address http://localhost:13060.
 
+### Local setup
+
+#### Requirements
+- A PostgreSQL installation (10.3, 9.6.8, 9.5.12, 9.4.17, 9.3.22 or newer)
```

Comment:
Have you checked it's working with 10.3 ? Last time I checked it was only working with 9.x

Reply:
I'll verify to be sure

Reply:
So, let's just say 9.x, 10.x or newer ?

Maybe add Python there as a requirement?

Reply:
I explicitly added these versions because the seed db can't be loaded with lower versions than these because it was dumped with a version that had this security patch included. Lower PG versions won't be able to load the dump.
I'll add Python 2.7 as a requirement.

Reply:
Ok, thank you, get it.